cmd/gateway/s3x/gateway-s3x-utils.go

```diff
@@ -48,4 +56,91 @@ func pingHash(hash string) {
 			log.Println(fmt.Sprintf("pinged to url gateway %s with hash %s", url, hash))
 		} (url)
 	}
+}
+
+// API is a property on handlerinput.entry
+type API struct {
```

Comment:
i think we can move all this to its own go file to separate our fleek stuff from the s3x utils